docs/workflows.rst

```diff
@@ -575,11 +575,6 @@ Confounds estimation
     wf = init_pet_confs_wf(
         name="discover_wf",
         mem_gb=1,
-        metadata={
-            "FrameTimesStart": [0, 2, 4, 6],
-            "FrameDuration": [2, 2, 2, 2],
```

Comment:
@mnoergaard These seems like should be used, but they are not as far as I understand.

Reply:
This seems to be dead code